common/app/rendering/core/JavascriptProps.scala

```diff
@@ -0,0 +1,16 @@
+package rendering.core
+
+import conf.Configuration
+import play.api.libs.json.{Json, Writes}
+
+case class JavascriptProps(
+  bundleUrl: String = conf.Static("javascripts/ui.bundle.browser.js"),
```

Comment:
My understanding (please correct me if this isn't the best way anymore) is that the props would look something like this:
```
{
  path( or another name ): String,
  config: {...}
  content: {...}
  ...
}
```
Certainly `bundleUrl`. `polyfillioUrl` and definitely `beaconUrl` probably belong in `config` , and not high level on their own.

What do you think @SiAdcock ?

Reply:
Definitely agree that all these properties belong in a `config` object, rather than at this top level. I shall look to move them down.

`path` would be useful for telling the `ui` app which component or page to render. `content`... I'm not sure how I'd define an arbitrary object like this in Scala! I'll pair with someone when I'm back in the office. 

I'm going to add a task to the Trello board to formalise the API for communication between Play and JS rendering. This is a good starting point.